api/Insight_API_documentation.md

```diff
@@ -0,0 +1,1198 @@
+                        "asm": "3045022100b683ac2b7668840dc048944407aa941e8f2e026f1d3a708bf7dfe0eea799700502206488d9da5301893672b97412f6dd27657c108682c6b7feefca15f45f0e739b4501 02f8857d1459fb56fe3ba0f006943f2ec785f5e4bbd1d2480d0a1af2e8875214da"
+                    },
+                    "addr": "Dsbr3zGyBcDYWWibnENAhW3N8UwJ5J4tnKg",
+                    "valueSat": 8965893473,
+                    "value": 89.65893473
+                },
+.
+.
+.
+.
+
+            ],
+            "blockhash": "0000000000000008537e82b35ac3e7b4549289cfb440e1218260ef3fceb7770e",
+            "blockheight": 229806,
+            "confirmations": 49336,
+            "time": 1523675017,
+            "blocktime": 1523675017,
+            "valueOut": 4248.47935122,
+            "size": 10043,
+            "valueIn": 4248.48951622,
+            "fees": 0.010165
+        },
+
+    ]
+}
+```
+
+<br/>
+
+### /tx/send/ (POST)
+
+**URL:**  ```POST /tx/send ``` 
+
+**Description:** Broadcasts transaction to network.
+
+**Parameters:**
+
+| Parameter           | Type                   |  Description                   | 
+| -------------------- | ---------------------- | ---------------------- | 
+| rawtx              | `string`      |   Signed transaction as hex string       |  
+
+
+**Request Example:**
+
+```
+curl -X POST \
+ https://alpha.dcrdata.org/insight/api/tx/send \
+ -H 'Cache-Control: no-cache' \
+ -H 'Content-Type: application/json' \
+ -H 'Postman-Token: bf2e16a1-b6d2-4b2f-b6e5-9ea7bd3df5b1' \
+ -d '{"rawtx":"010000000184de71690b97b4cbac6e723570a25a56295b30aa26345bfc40c0609c87e23f1d0100000000ffffffff02809698000000000000001976a9143eb656115197956125365348c542e37b6d3d259988ac00811b2c0000000000001976a914e4c9ada86ba67b2c082fac1aa09fb17fccc6833688ac000000000000000001ffffffffffffffff00000000ffffffff6b483045022100b5020004e60a4d26c99a00316ed4f51131b99838fd8fab936d55f9475719bb3b022063b86596adba004367a138f62c73d59f8f3b2cded4b10b5fc0ebe92a4cf41c840121029ec6a82a9646c090decf20806029f332cc11a5c6fa17c943ee72c31b9707a433"}'
+```
+**Request Response:**
+```
+{
+    "txid": "18a4eeed058c2266512863d03f79651cd38d94d6d682ae3f1e4aad0178c6998f"
+}
+```
+<br/>
+
+## Addresses 
+
+Methods that work with addresses.
+
+### /addr/
+
+**URL:**  ```GET /addr/{address}```
+
+**Description:** Retrieves transactions by addresses. Can optionally return transactions from `N` to `M`.
+
+**Parameters:**
+
+| Parameter           | Type                   |  Description                   | 
+| -------------------- | ---------------------- | ---------------------- | 
+| hash              | `string`      |   Address    |  
+| N              | `int64`      |   (optional) Starting transaction index    |  
+| M              | `int64`      |   (optional) Ending transaction index    |  
+| noTxList              | `boolean`      |   (optional) If `noTxList` = '1', response will not include a list of txids     |  
```

Comment:
same with this one.  I think the parameters are `address`, `from`, `to` (and noTxList as you already have) based on the example below